spec/section/target-vocabulary.md

```diff
@@ -106,200 +106,18 @@ If unspecified, the default value is UTF-8.
 This namespace is NOT limited to the listed compression formats 
 and MAY be extended in the future.
 
-#### Relative paths
```

Comment:
See aslo my comment https://github.com/kg-construct/rml-core/pull/152. I think this section should remain here.